src/SplitPackages/SplitPackages.csproj

```diff
@@ -0,0 +1,50 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <Import Project="..\..\build\common.props" />
+  <PropertyGroup>
+    <Description>Copies NuGet packages form a given source folder into a specific set of folders based on a CSV file.</Description>
+    <VersionPrefix>1.0.1</VersionPrefix>
+    <TargetFrameworks>netcoreapp1.0;net451</TargetFrameworks>
```

Comment:
Could you create a follow up to stop cross compiling this? There's seemingly only one location which runs this (https://github.com/aspnet/Coherence/blob/8f8c30b5b57e9171b444c9b505db70386a17a9f3/makefile.shade#L53) and it should be able to run it as a CoreCLR app there.